javascript/usearch.ts

```diff
@@ -351,8 +354,14 @@ export class Index {
       );
     }
 
+    if ((!Number.isNaN(threads) && typeof threads !== "number") || threads < 0) {
```

Comment:
Since the validation of `threads` is the same, it would be better to make it a method.
I will refactor it if you like.